diff -urN mailfilter-0.5.1.orig/configure.ac mailfilter-0.5.1/configure.ac
--- mailfilter-0.5.1.orig/configure.ac	2003-01-01 20:25:10.000000000 +0100
+++ mailfilter-0.5.1/configure.ac	2003-07-24 00:02:37.000000000 +0200
@@ -69,7 +69,7 @@
 AC_HEADER_TIME
 
 # GNU gettext.
-ALL_LINGUAS="de el es fr it pt_BR ru"
+ALL_LINGUAS="de el es fr it pl pt_BR ru"
 AM_GNU_GETTEXT
 AM_GNU_GETTEXT_VERSION(0.11.5)
 

